source/core/slang-performance-profiler.cpp

```diff
@@ -65,7 +65,7 @@ namespace Slang
 
             if (strSize > 0)
             {
-                strncpy_s(profileEntry.funcName, strSize, func.key, strSize);
+                memcpy(profileEntry.funcName, func.key, strSize);
             }
```

Comment:
Also note that when `strlen()` returns zero, you need to copy a null terminator.
The current code will unchange the name, which is probably not what is indended.